lightning/src/onion_message/functional_tests.rs

```diff
@@ -111,16 +111,37 @@ impl Writeable for TestCustomMessage {
 }
 
 struct TestCustomMessageHandler {
-	expected_messages: Mutex<VecDeque<TestCustomMessage>>,
+	expected_messages: Mutex<VecDeque<Expectations>>,
```

Comment:
Let's rename the field `expectations`

Reply:
Done! [e6effd4](https://github.com/lightningdevkit/rust-lightning/commit/e6effd447d47e1c3c7771935bea58801199b60b3)
Thanks!